addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -162,20 +162,32 @@ def get_days(self, employee_id):
 
         for request in requests:
             status_dict = result[request.holiday_status_id.id]
-            status_dict['virtual_remaining_leaves'] -= request.number_of_days
+            status_dict['virtual_remaining_leaves'] -= (request.number_of_hours_display
```

Comment:
The `number_of_hours_display` field is intended for display only, not computation. Not sure you should use it here.